<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolverSetupWidget</class>
 <widget class="QWidget" name="SolverSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbGetAll">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSetAll">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Solver Commands</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Solver Commands</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="3" rowspan="5">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QComboBox" name="cbClientType">
              <item>
               <property name="text">
                <string>Administrator</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Operator</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>DataSender</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lePassword">
              <property name="echoMode">
               <enum>QLineEdit::PasswordEchoOnEdit</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbSetUser">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0" rowspan="2" colspan="3">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="2" column="1">
             <widget class="QPushButton" name="pbCorReset">
              <property name="text">
               <string>Reset Corrections</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pbCorSave">
              <property name="text">
               <string>Save Corrections</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pbStop">
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pbRestart">
              <property name="text">
               <string>Restart</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pbClear">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pbCorStart">
              <property name="text">
               <string>Start Corrections</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pbCorStop">
              <property name="text">
               <string>Stop Corrections</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Solver Params</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Solver Params</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Max. SKO</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Max. Position error</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Trajectory length:</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1" rowspan="3" colspan="2">
           <widget class="QTableWidget" name="twWorkRect">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>280</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::NoDragDrop</enum>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <property name="rowCount">
             <number>2</number>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <row>
             <property name="text">
              <string>Left down</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>Right up</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>lon</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>lat</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Aim height:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="cbAuto">
            <property name="text">
             <string>Auto</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>SolveTypes</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QCheckBox" name="cbTrajectories">
               <property name="text">
                <string>Trajectories</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="cbOneMarks">
               <property name="text">
                <string>One Marks</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="cbParticles">
               <property name="text">
                <string>Particles</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="cbPositionLines">
               <property name="text">
                <string>Position lines</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Max. speed</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QSpinBox" name="sbTrajectoryLen">
            <property name="suffix">
             <string> sec</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>60000</number>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>From map</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Trajectory points:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QSpinBox" name="sbTrajectoryPoints">
            <property name="suffix">
             <string> points</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QSpinBox" name="sbPositionError">
            <property name="suffix">
             <string> m</string>
            </property>
            <property name="minimum">
             <number>500</number>
            </property>
            <property name="maximum">
             <number>15000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Solver Area</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QSpinBox" name="sbAimHeight">
            <property name="suffix">
             <string> m</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QSpinBox" name="sbSpeed">
            <property name="suffix">
             <string> km/h</string>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLabel" name="lblSolverVersion">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Min delays for single marks</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QSpinBox" name="sbSKO">
            <property name="suffix">
             <string> m</string>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="cbManual">
            <property name="text">
             <string>Manual</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1" colspan="2">
           <widget class="QSpinBox" name="sbMinDelays">
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Merge</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QCheckBox" name="cbIsMerge">
            <property name="text">
             <string>isMerge</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <widget class="QDoubleSpinBox" name="sbMergeTime">
            <property name="suffix">
             <string> sec</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Stations and Corrections</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="twStations">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>500</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <column>
          <property name="text">
           <string>Station</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Lon</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Lat</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Height</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>Click on map</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Stations Coordinates:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QTableWidget" name="tableWidget_3">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>60</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <row>
          <property name="text">
           <string>Iri</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Lat</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Lon</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Height</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1" colspan="4">
        <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>Click on map</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>67</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Corrections</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Accurancy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
